content/en/docs/components/multi-tenancy/overview.md

```diff
@@ -45,7 +41,10 @@ controlled by the profile access policies set by the administrator or the owners
 of the profiles. Resources created by the notebooks (for example, training jobs and
 deployments) also inherit the same access.
 
-Metadata and Pipelines or any other applications currently don't have full
+Pipelines is partially integrated with multi-user isolation starting from Kubeflow v1.1. You
+can find more information on [Multi-user Isolation for Pipelines](https://www.kubeflow.org/docs/pipelines/multi-user/).
+
```

Comment:
Hey @Bobgy Thank you! Just a small suggestion: "Pipelines is..." -> "Kubeflow Pipelines is..." to emphasize that it's KFP we're talking about, not just some ML "pipelines". I wrapped the text for you, just in case:

```suggestion
Kubeflow Pipelines is partially integrated with multi-user isolation starting
from Kubeflow v1.1. You can find more information on [Multi-user Isolation for
Pipelines](https://www.kubeflow.org/docs/pipelines/multi-user/).

```